content/03.methods.md

```diff
@@ -5,8 +5,72 @@ Talk about dataset - Pubtator
 Talk about preprocessing Pubtator
 Talk about hand annotations for each realtion
 
-## Label Functions
-describe what a label function is and how many we created for each relation
+## Label Functions for Annotating Sentences
+A common challenge in natural language processing is having too few ground truth annotations, even when data are abundant.
```

Comment:
possible to replace "data" with something more specific... are you referring to "input text" or the like?